src/IO/CFS/WriteBufferFromCFS.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "config.h"
+#include <IO/WriteBuffer.h>
+#include <IO/WriteSettings.h>
+#include <IO/WriteBufferFromFileBase.h>
+#include <Poco/Util/AbstractConfiguration.h>
+#include <fcntl.h>
+#include <string>
+#include <memory>
+
+
+#ifndef O_DIRECT
+#define O_DIRECT 00040000
+#endif
+
+namespace DB
+{
+/**
+ * Accepts CFS path to file and opens it.
+ * Closes file by himself (thus "owns" a file descriptor).
+ * The NFS protocol is compatible with most POSIX interface,
+ * but there are still differences in performance and local file systems.
+ */
+class WriteBufferFromCFS final :  public WriteBufferFromFileBase
```

Comment:
What the difference actually is?
If it is mostly copy-paste of WriteBufferFromFile, can we unify it?